Makefile

```diff
@@ -305,8 +305,7 @@ protoImage=$(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace $(pro
 
 proto-swagger-gen:
 	@echo "Generating Protobuf Swagger"
-	#@$(protoImage) sh ./proto/scripts/proto-swagger-gen.sh
-	./proto/scripts/proto-swagger-gen.sh
+	@$(protoImage) sh ./proto/scripts/proto-swagger-gen.sh
```

Comment:
can you make another command that directly does from scripts :P 
proto-swagger-gen-direct? or something

Reply:
okie